internal/acctest/acctest.go

```diff
@@ -24,6 +26,16 @@ var (
 	}
 )
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyz"
+
+func RandomNameWithPrefix(prefix string) string {
+	b := make([]byte, 6)
+	for i := range b {
+		b[i] = letterBytes[rand.Intn(len(letterBytes))]
+	}
+	return fmt.Sprintf("%s-%s", prefix, b)
+}
+
```

Comment:
I see it being used to add unique joint token names for each of the acceptance tests. If so, can you add a function comment?

Reply:
This will be used for all resources where we use unique names. Even the IP space one, I have to go back and change it at some point.

Reply:
Will add a function comment though